debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerChangeTable.java

```diff
@@ -31,19 +36,19 @@
      */
     private Lsn stopLsn;
 
-    /**
-     * The table from which the changes are captured
-     */
-    private Table sourceTable;
+    private List<String> capturedColumnList;
```

Comment:
Can be `final`? Also simply `capturedColumns` would be preferred, no need to encode the type into the name (same for the getter exposing it).

Reply:
Yeah, I can make that change. I had both the original string and a list at one point but with the string gone, you're right the "list" is unnecessary.